notebook/performance_demo.ipynb

```diff
@@ -0,0 +1,312 @@
+     "text": [
+      "     Unnamed: 0                                            message\n",
+      "0           242  Von meinem iPhone gesendet Anfang der weiterge...\n",
+      "1           243  Von meinem iPhone gesendet Anfang der weiterge...\n",
+      "2           244  Von meinem iPhone gesendet Anfang der weiterge...\n",
+      "3           245  Von meinem iPhone gesendet Anfang der weiterge...\n",
+      "4           246  Von meinem iPhone gesendet Anfang der weiterge...\n",
+      "..          ...                                                ...\n",
+      "98         1313  \\nVon: Mélissa des Presses de l'Université Lav...\n",
+      "99         1314  Von: Librairie Classiques Garnier &amp;lt;libr...\n",
+      "100        1315  La langue s'enrichit #36 - FranceTerme\\nProf. ...\n",
+      "101        1316  Activités de juin - Presses de l'Université La...\n",
+      "102        1317  Nouveautés de juin\\nProf. Dr. Sybille Große I ...\n",
+      "\n",
+      "[103 rows x 2 columns]\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Some weights of the model checkpoint at xlm-roberta-large-finetuned-conll03-english were not used when initializing XLMRobertaForTokenClassification: ['roberta.pooler.dense.bias', 'roberta.pooler.dense.weight']\n",
+      "- This IS expected if you are initializing XLMRobertaForTokenClassification from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).\n",
+      "- This IS NOT expected if you are initializing XLMRobertaForTokenClassification from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).\n"
+     ]
+    }
+   ],
+   "source": [
+    "# import files from csv file\n",
+    "email_list = pd.read_csv(\"../mailcom/test/data/mails_lb_sg.csv\")\n",
+    "print(email_list)\n",
+    "\n",
+    "# create pseudonymization object\n",
+    "ps = mailcom.parse.Pseudonymize()\n",
+    "ps.init_spacy(\"fr\")\n",
+    "ps.init_transformers()\n",
+    "# time stamp after model loading\n",
+    "t_model_loaded = time.time()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# loop over mails and pseudonymize them\n",
+    "out_list = []\n",
+    "ts_list = []\n",
+    "for idx, row in email_list.iterrows():\n",
+    "    text = row[\"message\"]\n",
+    "    email_dict = {\"content\": text}\n",
+    "    if not text:\n",
+    "        continue\n",
+    "    # Test functionality of Pseudonymize class\n",
+    "    output_text = ps.pseudonymize(text)\n",
+    "    email_dict[\"pseudo_content\"] = output_text\n",
+    "    out_list.append(email_dict)\n",
+    "\n",
+    "    # timestamp after this email\n",
+    "    ts_list.append(time.time())"
```

Comment:
Can we add something to plot the timestamps and get the average? And include the time for model loading / preprocessing.
For example, two bar plots:
- One with the time per email, to see if there are any emails that take longer. 
- One with the time for model loading, time for preprocessing (reading the eml files and spacy sentencizing), and average time of processing per email (if that is legible on the same plot).